components/usage/pkg/apiv1/billing.go

```diff
@@ -32,8 +32,10 @@ func NewBillingService(stripeClient StripeClient, billInstancesAfter time.Time,
 }
 
 type StripeClient interface {
-	GetUpcomingInvoice(ctx context.Context, kind stripe.CustomerKind, id string) (*stripe.StripeInvoice, error)
+	GetUpcomingInvoice(ctx context.Context, customerID string) (*stripe.Invoice, error)
 	UpdateUsage(ctx context.Context, creditsPerTeam map[string]map[string]float64) error
+	GetCustomerByTeamID(ctx context.Context, teamID string) (*stripe.Customer, error)
+	GetCustomerByUserID(ctx context.Context, userID string) (*stripe.Customer, error)
```

Comment:
After merging changes, this `*stripe.Customer` gets undeclared errors inexplicably 😵